nitrorole/nitrorole.py

```diff
@@ -317,6 +317,42 @@ async def cog_disabled_in_guild(self, guild: Optional[discord.Guild]) -> bool:
             return False
         return await func(self, guild)
 
+    @nitrorole.command(name="settings", aliases=["show", "showsettings", "setting"])
+    async def nitrorole_settings(self, ctx: GuildContext) -> None:
+        """
+        Shows the current NitroRole settings for the server.
+        """
+        
+        guild = ctx.guild
+        guild_data = await self.get_guild_data(guild)
+        role = guild_data.role_id
+        channel_id = guild_data.channel_id
+        unassign_on_boost_end = guild_data.unassign_on_boost_end
+
+        # Check if the channel is set, implying the cog is enabled
+        is_enabled = "Enabled" if channel_id else "Disabled"
+
+        role_mention = f"<@&{role}>" if role else "None"
+        channel = guild.get_channel(channel_id)
+        channel_mention = channel.mention if channel else "None"
```

Comment:
I have a stylistic preference for using `is None` over truthiness.
```suggestion
        channel_mention = channel.mention if channel is not None else "None"
```